projects/example-app/src/app/core/services/book-storage.service.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { TestBed } from '@angular/core/testing';
+import {
+  BookStorageService,
+  IBookStorageService,
+  LOCAL_STORAGE_TOKEN,
+} from './book-storage.service';
+import { cold } from 'jasmine-marbles';
+import { Book } from '@example-app/books/models/book';
+
+describe('BookStorageService', () => {
+  let fixture: IBookStorageService;
+
+  let localStorageFake: Storage & any = {
+    removeItem: jest.fn(),
+    setItem: jest.fn(),
+    getItem: jest.fn(_ => JSON.stringify(persistedCollection)),
+  };
+
+  const book1 = { id: '111', volumeInfo: {} } as Book;
+  const book2 = { id: '222', volumeInfo: {} } as Book;
+  const book3 = { id: '333', volumeInfo: {} } as Book;
+  const book4 = { id: '444', volumeInfo: {} } as Book;
+
+  const persistedStorageKey = 'books-app';
+  const persistedCollection = [book2, book4];
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        {
+          provide: LOCAL_STORAGE_TOKEN,
+          useValue: localStorageFake,
+        },
+      ],
+    });
+    fixture = TestBed.get(BookStorageService);
+  });
+  describe('supported', () => {
```

Comment:
Nit, add a line break between `describe`s / `it`s

Reply:
Formatting is default `vscode` i believe, it also matches all other `spec` formatting in the project?

Reply:
I'm not sure if VSCode does this 🤔

Reply:
Apologies you mentioned between `describe`s / `it`s, which it must do as I use a vanilla install of `vscode` with default settings, unless its the `prettier` formatter? if it was between the `beforeEach` and `describe` you intended then i have already actioned. I also believe the project runs a pre-commit formatting task.